### PR TITLE
Notice other versions of the same page

### LINKED ISSUE #193: Show notice about another similar pages
For instance there are two pages with the same name in `common/` and `linux`. Before each page in output we can write smth like:

- `There are another page versions in: common.` (for `linux/`)
- `There are another page versions in: linux.` (for `common/`)

Note that difference between these pages are:

- `common/` uses short options for the most frequently used arguments and long for others
- `linux/` uses long options for all situations

### PR DESCRIPTION
**Pull request by Jaimepas77**
Fixes #193 

This PR changes the way pages are searched so that it can find multiple versions of the same page. Therefore, every time a page is searched, every possible platform is checked even if a previous valid one was found. If more than one page is found, the next platform in the priority list is noticed to the user.

It doesn't search in every page if the platform was determined by the user parameters.

The PR implements this [specification](https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#if-multiple-versions-of-a-page-were-found).

Bugs: it wont find multiple versions of the page if the cache only contains one of them. As it's much faster to search on the cache, this bug was not resolved.

### CODE REVIEW

File: tldr.py [hunk 1]
```diff
@@ -553,20 +606,23 @@ def main() -> None:
...
                     "`{cmd}` documentation is not available.\n"
                     "If you want to contribute it, feel free to"
                     " send a pull request to: https://github.com/tldr-pages/tldr"
                 ).format(cmd=command))
             else:
-                output(result, plain=options.markdown)
+                output(results[0][0], plain=options.markdown)
+                if results[1:]:
+                    platform_str = results[1][1]
+                    print(f"There are other versions of this page in the {platform_str} category.")
```
Comment (CleanMachine1) [inline tldr.py:625]:
```suggestion
                    print(f"There are other versions of this page under the {platform_str} platform.")
```

File: tldr.py [hunk 2]
```diff
@@ -553,20 +606,23 @@ def main() -> None:
...
                     "`{cmd}` documentation is not available.\n"
                     "If you want to contribute it, feel free to"
                     " send a pull request to: https://github.com/tldr-pages/tldr"
                 ).format(cmd=command))
             else:
-                output(result, plain=options.markdown)
+                output(results[0][0], plain=options.markdown)
+                if results[1:]:
+                    platform_str = results[1][1]
+                    print("There are other versions of this page under the {} platform.".format(platform_str))
```
Comment (vitorhcl) [inline tldr.py:625]:
```suggestion
                    platforms_str = [result[1] for result in results[1:]]
                    single = len(platforms_str) == 1
                    print(f"There {'is' if single else 'are'} other version{'' if single else 's'} of this page under the following platform{'' if single else 's'}: {', '.join(platforms_str)}.")
```

**Review by vitorhcl — CHANGES_REQUESTED** [review]

**Comment by vitorhcl** [comment]
I ran black on the script

**Comment by vitorhcl** [comment]
I reverted black formatting to not conflict with other PRs for now. We can maybe run it in another PR

**Review by kbdharun — APPROVED** [review]

**Review by vitorhcl — COMMENTED** [review]
LGTM after some minor changes

**Review by kbdharun — APPROVED** [review]

**Review by vitorhcl — APPROVED** [review]
LGTM, thank you @kbdharun and @Jaimepas77!

**Pull request merged**